SimG4CMS/Forward/src/ZdcShowerLibrary.cc

```diff
@@ -27,10 +26,31 @@ ZdcShowerLibrary::ZdcShowerLibrary(const std::string& name, edm::ParameterSet co
 
 ZdcShowerLibrary::~ZdcShowerLibrary() {}
 
-std::vector<ZdcShowerLibrary::Hit>& ZdcShowerLibrary::getHits(const G4Step* aStep, bool& ok) {
-  const G4StepPoint* preStepPoint = aStep->GetPreStepPoint();
-  const G4StepPoint* postStepPoint = aStep->GetPostStepPoint();
-  const G4Track* track = aStep->GetTrack();
+void ZdcShowerLibrary::initRun(G4ParticleTable* theParticleTable) {
+  G4String parName;
+  emPDG = theParticleTable->FindParticle(parName = "e-")->GetPDGEncoding();
```

Comment:
These PDG codes can be taken from PDG directly. The call to theParticleTable is not needed.